alert-common/src/main/java/com/synopsys/integration/alert/common/enumeration/AccessOperation.java

```diff
@@ -27,5 +27,8 @@
     DELETE,
     READ,
     WRITE,
-    EXECUTE
+    EXECUTE,
+    UPLOAD_FILE_READ,
```

Comment:
Is the reason that these are different because of the way we currently have custom endpoints set up?

Reply:
I didn't want to allow delete because if we allow delete on the component_settings global context then through the rest API you can delete the global component_settings configuration which could potentially break the authentication into the UI and being able to read and write encrypted data.

Reply:
Because of this I decided to make upload a special operation.